src/catalog/src/system_schema/pg_catalog.rs

```diff
@@ -93,6 +97,11 @@ impl PGCatalogProvider {
         for name in MEMORY_TABLES.iter() {
             tables.insert(name.to_string(), self.build_table(name).expect(name));
         }
+        tables.insert(
+            PG_NAMESPACE.to_string(),
+            self.build_table(PG_NAMESPACE).unwrap(),
+        );
+        tables.insert(PG_CLASS.to_string(), self.build_table(PG_CLASS).unwrap());
```

Comment:
try to avoid `unwrap` here. If it's really safe to use unwrap, leave a comment for the background

Reply:
Yeah. Should use `expect` instead.PG_NAMESPACE

Reply:
Oh wait, expect is actually same with unwrap and it's just carrying a text message for the panic. We need to avoid panic in worst case.

Reply:
Actually it's safe to use `unwarp` or `expect` here. If the constants passed into `build_table` have been handled by 'system_table`, it bounds to return a `SystemTableRef`. Since the function is not able to be called outside and the match logic is carefully checked, I think it's ok to unwrap here. Option is just used to pass type check here(since table_name is not modeling as a type). Even panic, it will just happen is dev stage. To cleanup the code, I would use `enum` and `strum` to statically trace the table supported to avoid `Option` here. This can be done by another pr.

Reply:
Sounds good. Please leave a comment and TODO here to other developers to understand this.